src/js/components/awardv2/financialAssistance/FinancialAssistanceContent.jsx

```diff
@@ -65,12 +66,18 @@ const FinancialAssistanceContent = ({ awardId, overview, jumpToSection }) => {
 
     const awardAmountData = Object.create(BaseAwardAmounts);
     awardAmountData.populate(overview, overview.category);
+    let title = overview.typeDescription;
+    if (overview.category === 'grant') {
+        const titleArray = title.split(' ').map((word) => upperFirst(word.toLowerCase()));
+        if (titleArray.length === 3) titleArray.pop();
```

Comment:
Why are we only using two of the words taken from the array?

Reply:
the last one is `(A)` or `(B)`.

Per this comment, `I don't think the letters add value at this point so would prefer just the letter-less title`, in [this ticket](https://federal-spending-transparency.atlassian.net/browse/DEV-3633)

Reply:
Nicely done. Can we add a comment in the code as well to this effect?

Reply:
@MaxwellKendall done